<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicatorSerialWidget</class>
 <widget class="QWidget" name="CommunicatorSerialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>310</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="settings">
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="portSelectiontreeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>241</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <column>
      <property name="text">
       <string>Devices / Channels</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UDP</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Selection</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="TargetstackedWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="serial_settings">
      <widget class="QGroupBox" name="StopBitsgroupBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>81</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Stop Bits</string>
       </property>
       <widget class="QRadioButton" name="stop1bitsRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="stop2bitsRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>61</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="ParitygroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Parity</string>
       </property>
       <widget class="QRadioButton" name="parityNRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>N</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="parityERadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>61</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>E</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="parityORadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>61</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>O</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="dialLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>BaudRate</string>
       </property>
      </widget>
      <widget class="QDial" name="BaudrateSpeedDial">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>50</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="SizegroupBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Size</string>
       </property>
       <widget class="QRadioButton" name="databits8radioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>8-bit</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="databits7radioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>7-bit</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="TCP">
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="serial_data">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QListWidget" name="SerialDisplaylistWidget"/>
      </item>
      <item>
       <widget class="QComboBox" name="SendInput">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCommandLinkButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="CloseCommandLinkButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
